src/main/java/seedu/trackermon/logic/commands/FindCommand.java

```diff
@@ -19,7 +19,7 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all shows whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "At least one parameter must be stated in the find command.\n"
-            + "Parameters: [NAME] [STATUS] [TAG]\n"
+            + "Parameters: [NAME] [STATUS] [RATING] [TAG]...\n"
```

Comment:
Should it be [t/TAG]… instead?

Reply:
Ah yes, I will add prefixes to all of them.